cypress/e2e/lightspeed-org-admin.cy.ts

```diff
@@ -0,0 +1,10 @@
+describe('Navigate to the Seats Administration page as lightspeed-test-user', () => {
```

Comment:
I description should be "as lightspeed-org-admin" no?

Reply:
yes - just fixed, thanks :D